client/assets/components/modalContainer.scss

```diff
@@ -0,0 +1,82 @@
+.modal {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background-color: rgba(0, 0, 0, 0.64);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  z-index: 1040;
+
+  .modal__content {
+    background-color: #fff;
+    padding: 25px 25px;
+    width: 90%;
+    max-width: 768px;
+    overflow-y: auto;
+    border-radius: rem(6px);
+
+    @media (min-width: 600px) {
+      padding: 60px 60px;
+      max-width: 634px;
```

Comment:
### Nitpick

Can we standardize the unit? Seeing rems and px, you can make use of your `px-to-rem` function for conversion